AutoscoperM/AutoscoperM.py

```diff
@@ -236,8 +236,10 @@ def onApplyButton(self):
         import sys  # noqa: F401
 
         if sys.platform == "darwin":
-            slicer.util.messageBox("AutoscoperM is not supported on this platform.<br>"
-                "See <a href='https://autoscoperm.slicer.org'>https://autoscoperm.slicer.org</a> for details.")
+            slicer.util.messageBox(
+                "AutoscoperM is not supported on this platform.<br>"
+                "See <a href='https://autoscoperm.slicer.org'>https://autoscoperm.slicer.org</a> for details."
+            )
```

Comment:
Since the line-length associated with `black` is only specified in .pre-commit-config.yaml, I suspect the your local development environment has a lower default for the line length.

https://github.com/BrownBiomechanics/SlicerAutoscoperM/blob/1a840deeb03a5f7936a26653e60edf2a58bba542/.pre-commit-config.yaml#L5-L6